libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2088,42 +2088,6 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
       },
     ],
   },
-  pivx: {
```

Comment:
should we remove the pivx here?
AFAIK, we remove pivx support from Ledger-live. but the pivx nano app is still there. 
i.e. we should give users a way to download the pivx nano app from Ledger Live manager.